src/server.js

```diff
@@ -0,0 +1,152 @@
+const _ = require('lodash/fp')
+const express = require('express')
+const cors = require('cors')
+const uuid = require('uuid/v4')
+const { google } = require('googleapis')
+const { Firestore } = require('@google-cloud/firestore')
+const bodyParser = require('body-parser')
+const Joi = require('@hapi/joi')
+const { promiseHandler, Response, validateInput } = require('./utils')
```

Comment:
That plugin tries to sort by variable name, rather than path, and doesn't seem to deal well with destructuring. I didn't see another alternative that deals with `require`

Reply:
True enough. We should at least sort manually.